cobra/cmd/licenses.go

```diff
@@ -41,6 +41,7 @@ type License struct {
 func init() {
 	// Allows a user to not use a license.
 	Licenses["none"] = License{"None", []string{"none", "false"}, "", ""}
+	Licenses["empty"] = License{"Empty", []string{"empty", "blank"}, "", ""}
```

Comment:
New License type. Also set as the default type.
None License type used to omit License creation.